global_planner/src/nodes/global_planner_node.cpp

```diff
@@ -390,9 +381,7 @@ void GlobalPlannerNode::plannerLoopCallback(const ros::TimerEvent& event) {
   }
 
   // If the current cell is blocked, try finding a path again
```

Comment:
This comment doesn't make sense any more

Reply:
I have deleted the comment